### DISCUSSION TITLE
Chrome 131 results

### DISCUSSION THREAD

**Discussion by krausest** [krausest/js-framework-benchmark#1782] [Announcements]
Results for chrome 131 are [published](https://krausest.github.io/js-framework-benchmark/2024/table_chrome_131.0.6778.85.html).

We're using Chrome for Testing 131.0.6778.85.html for this run and it seems to be a pretty boring result regarding this benchmark.

Attached you find a binary build of all frameworks such that you can run the benchmark without installing and building all frameworks (see README for how to use the build.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/krausest/js-framework-benchmark/releases/tag/chrome131'>Chrome 131 results</a>.</em>

**Comment by syduki** [comment]
> it seems to be a pretty boring result

May I ask why? I find it pretty exciting that `svelte` knocked out `solid` :laughing:. But seriously, there is a significant performance increase in 'replace rows' across nearly all frameworks, for some it is more than 20%, that's crazy figures, some even became faster in 'replace' than in 'create.'. That is a pretty impressive performance boost I've seen in a while.
